src/index.ts

```diff
@@ -18,14 +18,11 @@ const renderUsers = async (config: ApiConfig) => {
 };
 
 const app = () => {
-  const config = (window as any).__CONFIG__;
-  delete (window as any).__CONFIG__;
```

Comment:
you can register config right here - `ioc.register('config', config.api);`, so no need to move the code.